.travis.yml

```diff
@@ -7,22 +7,22 @@ language: none
 matrix:
   include:
     - compiler: gcc
```

Comment:
Because the goal of this PR is to make it compile with default libraries in Trusty, when __build-essential__ and __qtbase5-dev__ are installed.